src/Viz.js

```diff
@@ -362,6 +373,7 @@ export default class Viz extends BaseClass {
 
     // Resets margins
     this._margin = {bottom: 0, left: 0, right: 0, top: 0};
+    this._legendMargin = {bottom: 0, left: 0, right: 0, top: 0};
```

Comment:
Because I'm not asking for this property to also be applied to the colorScale, controls, and timeline, let's rename it `this._padding`? That means we will be taking over the previous value, which I see is being used in back, title, and total. Remove those references, and add padding to the textBox config used for each component.